guides/doc-Project_API/topics/con_api-request-composition.adoc

```diff
@@ -0,0 +1,38 @@
+[id="api-request-composition"]
+= API request composition
+
+The built-in API reference shows the API route, or path, preceded by an HTTP method:
+
+----
+HTTP_METHOD API_ROUTE
+----
+
+To work with the API, construct a command using the `curl` command syntax and the API route from the reference document:
+[options="nowrap", subs="+quotes,attributes"]
+----
+$ curl --request _HTTP_METHOD_ \                   <1>
+--insecure \                                   <2>
+--user _sat_username:sat_password_ \             <3>
+--data @_file_.json \                            <4>
+--header "Accept:application/json" \        <5>
+--header "Content-Type:application/json" \      <5>
+--output _file_                                  <6>
+_API_ROUTE_ \                                    <7>
+| python3 -m json.tool                          <8>
+----
+
+<1> To use `curl` for the API call, specify an HTTP method with the `--request` option.
+For example, `--request POST`.
+<2> Add the `--insecure` option to skip SSL peer certificate verification check.
+<3> Provide user credentials with the `--user` option.
+<4> For `POST` and `PUT` requests, use the `--data` option to pass JSON formatted data.
+For more information, see xref:sect-API_Guide-Passing_JSON_Data_with_the_API_Request[].
+<5> When passing the JSON data with the `--data` option, you must specify the following headers with the `--header` option.
+For more information, see xref:sect-API_Guide-Passing_JSON_Data_with_the_API_Request[].
+<6> When downloading content from {ProjectServer}, specify the output file with the `--output` option.
+<7> Use the API route in the following format: `https://_{foreman-example-com}_/katello/api/activation_keys`.
```

Comment:
This example looks like it's Katello-specific. OK for now. Technically not ideal for foreman-{deb,el}.

Reply:
Perhaps I should mark the guide as "not ready" for foreman-{deb,el}?